deeplake/api/dataset.py

```diff
@@ -1273,6 +1292,7 @@ def ingest_yolo(
     def ingest_classification(
         src: Union[str, pathlib.Path],
         dest: Union[str, pathlib.Path],
+        sample_compression: str = "auto",
         image_params: Optional[Dict] = None,
```

Comment:
Should be rename to primary_params, and docstring should be updated. 

"primary" refers to the non-annotation tensor.

Reply:
Hey, sorry for being inactive. Will continue working on it and thanks for the review :)